docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md

```diff
@@ -0,0 +1,134 @@
+
+_View PRs charitably and in the best possible light._
+
+### Reviewing Issues
+
+Issues are a front-line support system to help others struggling with Gatsby. These are people we absolutely want to help because they're struggling with something and we want to unblock them. They are our champions who care enough to open an issue and request our assistance in solving a problem. Excellent support turns them into fans, contributors, and friends. However, on the flip side issues can also sometimes be a little frustrating and a huge time sink. It's a lot of work to diagnose a bug, so if it feels like the requestor isn't providing helpful, actionable resources for you to best assist, kindly suggest that they [supply a reproduction](https://gatsby.dev/reproduction). This lets you maximize the amount of time you can help others who have done some up-front work and focus on the issues that can be most easily and quickly remediated.
+
+Always help those who are struggling, but insist that they also help you help them.
+
+Sweet! You have reviewed a few PRs, helped solve an issue or two, and possibly fixed and PR'd a small bug. A nice, enjoyable start to your day, and now you can feel like you've helped out the community while also not exhausting a ton of mental effort in doing so.
+
+## Middle of day to end of day (4-6 hours)
+
+Feeling pretty good! Community focused tasks make it clear that the community is, and will always be, a priority for Gatsby. However--we also want to measurably move the needle forward and make Gatsby better for everyone. This will often involve features and fixes that require a deep context and understanding of Gatsby internals. Oftentimes (perhaps most of the time) these will correspond to a roadmap item which will support eventual larger goals tying to an Objective and Key Result (OKR), Key Product Initiative (KPI), and/or another team at Gatsby, e.g. Cloud. These are features that require deliberate focus to think about, approach, and ship with minimal outside distractions. To this end, some have found it valuable to block off chunks of time in their calendar so that meetings can't be scheduled, Slack is in do not disturb, etc. If these measures help you to truly focus and ship impactful features, by all means--do so! All at Gatsby recognize the value of focus and how it can effectively drive key results and features.
```

Comment:
I had thought KPI stood for Key Performance Indicator, though both sound pretty similar. I could be wrong 🤔

Reply:
@gillkyle is correct :)

Reply:
This is the problem with acronyms 😅 will fix! Thanks Kyle and Sam!

Reply:
Fixed :)